libp2p/protocols/pubsub/rpc/message.nim

```diff
@@ -53,7 +53,7 @@ proc verify*(m: Message, p: PeerInfo): bool =
 
 proc newMessage*(p: PeerInfo,
                  data: seq[byte],
-                 name: string,
+                 topic: string,
```

Comment:
btw, why a single topic here if the message can take multiple?

Reply:
yeah, this is't actually used, I considered removing it and I probably should.